### PR TITLE
Add name search on incoterm to search by code

### PR DESCRIPTION
**Pull request by yvaucher**

### CODE REVIEW

File: logistic_requisition/model/stock.py [hunk 1]
```diff
@@ -46,3 +47,15 @@ def name_get(self, cr, user, ids, context=None):
             name = "%s - (%s)" % (rs.code, rs.name)
             res += [(rs.id, name)]
         return res
+
+    @api.model
+    def name_search(self, name, args=None, operator='ilike', limit=100):
+        """
```
Comment (pedrobaeza) [inline logistic_requisition/model/stock.py]: Make it a one-line docstring.

File: logistic_requisition/model/stock.py [hunk 2]
```diff
@@ -46,3 +47,15 @@ def name_get(self, cr, user, ids, context=None):
...
+        Allows to search incoterms by code
+        """
+        if name:
+            recs = self.search([('code', operator, name)], limit=limit)
+            if recs:
+                return recs.name_get()
```
Comment (pedrobaeza) [inline logistic_requisition/model/stock.py]: Don't return this, but aggregate to other possible values from the name search, respecting limit variable. The best option for this is to combine all criteria in the search.

**Comment by yvaucher** [comment]
@pedrobaeza Fixed

**Comment by pedrobaeza** [comment]
Thanks.

:+1:

**Comment by gurneyalex** [comment]
:+1: very nice implementation

**Comment by lepistone** [comment]
cool, thanks :+1:

**Pull request merged**